pages/linux/pdftoppm.md

```diff
@@ -0,0 +1,24 @@
+# pdftoppm
+
+> Convert PDF document pages to Portable Pixmap (image formats).
```

Comment:
```suggestion
> Convert PDF document pages to portable Pixmap (image formats).
```
should this be capital?

Reply:
@olgashi you didn't answer the question or applied the change.